docker-compose.dev.yml

```diff
@@ -38,13 +38,13 @@ services:
     image: synadia/nats-server:nightly
     volumes:
       - ./config/:/etc/nats
-    command: "-c /etc/nats/nats.conf"
+    command: '-c /etc/nats/nats.conf'
     networks:
       - main
     ports:
-      - "4222:4222"
-      - "8222:8222"
-      - "9222:9222"
+      - '4222:4222'
+      - '8222:8222'
+      - '9222:9222'
```

Comment:
I think we should all have the same autoformatting configs so all code styling will be similar, for example now in my case autosave replaces double quotes with single quotes, and yours doing the opposite.

Reply:
yeah I agree with you but in this case the prettier config [rule](https://github.com/energywebfoundation/iam-cache-server/blob/6ed1cea58471fa4c6a765d13bfba94c8a9c84c53/.prettierrc#L2) in this repo is set to single quotes. i'm guessing the files were created before the rules were added.

Reply:
ok thank you.